app/src/main/java/com/kylecorry/trail_sense/tools/metaldetector/ui/FragmentToolMetalDetector.kt

```diff
@@ -80,6 +90,15 @@ class FragmentToolMetalDetector : BoundFragment<FragmentToolMetalDetectorBinding
                 isMetalDetected.debounceTime = Duration.ofMillis(100)
             }
         }
+
+        binding.metalSoundToggle.setOnCheckedChangeListener { _, isChecked ->
```

Comment:
Can this be moved to a toolbar button rather than be a switch?

Similar to the graph button on the battery tool - except this will act as a toggle instead

Reply:
That also means the label for this can be removed

Reply:
@kylecorry31 not sure I got this part... battery tools don't have tool bar but button

<img width="364" alt="Screenshot 2024-03-15 at 20 46 43" src="https://github.com/kylecorry31/Trail-Sense/assets/26343440/dcaffbe2-9cc0-42d2-9d1a-cf9aa7bc1216">

Reply:
Hi, I meant the buttons in the title bar at the top (history and settings): https://github.com/kylecorry31/Trail-Sense/blob/93cdb0e16ae0db96e86892c2e553bdfce7c43ce9/app/src/main/java/com/kylecorry/trail_sense/tools/battery/ui/FragmentToolBattery.kt#L127

It should use a sound icon and the button should act as a toggle (using CustomUiUtils.setButtonState to indicate if it is on or off)

Reply:
ok got this  part